kalite/main/tests/api_tests.py

```diff
@@ -1,9 +1,43 @@
 """
 """
+import json
+from django.core.urlresolvers import reverse
+
 from ..models import VideoLog, ExerciseLog
 from kalite.facility.models import Facility, FacilityUser
-from kalite.testing.base import KALiteTestCase
+from kalite.testing.base import KALiteTestCase, KALiteClientTestCase
 from kalite.testing.client import KALiteClient
+from kalite.topic_tools.content_models import set_database, Using, Item
+
+
+class ContentItemApiViewTestCase(KALiteClientTestCase):
+    @set_database
+    def setUp(self, db=None):
+        item_attributes = self.item_attributes = {
+            "title": "My Cool Item",
+            "description": "A description!",
+            "available": True,
+            "kind": "Video",
+            "id": "3",
+            "slug": "my_content",
+            "path": "khan/my_content",
+            "extra_fields": json.dumps({}),
+            "size_on_disk": 0,
+            "sort_order": 0,
+            "remote_size": 0
+        }
+        with Using(db, [Item], with_transaction=False):
+            self.item = Item.create(**item_attributes)
```

Comment:
@rtibbles I still expect this test to fail, but I'm not really sure how create a "real" content item. The one I create here is not returned by the content_item endpoint. Is this documented anywhere?